entrypoint.bash

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -eu
+
+export PATH=/usr/bin:/bin:/usr/sbin:/sbin
+
+tuser=tsvc
+name=telemetry-server
+_localstatedir=/var
+_tsvcdir=${_localstatedir}/lib/${tuser}
+_certdir=${_tsvcdir}/certs
+
+[ -d ${_certdir} ] && cp -a ${_certdir}/* /etc/pki/trust/anchors
+
+update-ca-certificates
+
+#For now commenting this out until we have a telemetry-server package built
```

Comment:
Not sure the comment directly relates to what the commented out code block does...

The code block is about checking to make sure that the service directory exists, with the expectation that if it doesn't exist it means we haven't mapped it in from the host environment...

However, for the telemetry server the expectation is that the service account directory would be provided by a mounted volume, to provide persistence of data, and that expectation is not dependent on whether the install is package based or not.

```suggestion
#For now commenting this out until figure out persistent volume mounts
```